backend/Squirrel.Core/Squirrel.Core.BLL/Services/DBStructureSaverService.cs

```diff
@@ -15,9 +14,9 @@ public DBStructureSaverService(IHttpClientService httpClientService, IConfigurat
         _configuration = configuration;
     }
 
-    public async Task SaveDBStructureToAzureBlob(ChangeRecord changeRecord)
+    public async Task SaveDBStructureToAzureBlob(ChangeRecord changeRecord, Guid clientId)
     {
-        await _httpClientService.PostAsync<Guid, Blob>
-            ($"{_configuration["SqlServiceUrl"]}/api/Changes", changeRecord.UniqueChangeId);
+        await _httpClientService.PostAsync<Guid, object>
```

Comment:
why `object`?

Reply:
Used to temporary avoid specifying TResponse and forgot about it) Thanks for noticing, fixed that